azext_edge/edge/_help.py

```diff
@@ -586,9 +586,19 @@ def load_iotops_help():
             data_source={data_source} name={name} observability_mode={observability_mode} sampling_interval={sampling_interval}
             queue_size={queue_size} --data data_source={data_source}
 
+        - name: Create an asset using a file containing data-points and another file containing events.
+          text: >
+            az iot ops asset create --name {asset_name} -g {resource_group} --custom-location {custom_location}
+            --endpoint {endpoint} --data-file {data_point_file_path} --event-file {event_file_path}
+
+        - name: Create an asset using a file containing data-points and another file containing events.
```

Comment:
With this example, I don't see why we need the previous example with placeholders.

Reply:
I thought there was some magical properties regarding the params in {} to be used by some help AI... but it does not matter much to me